include/psen_scan_v2/ros_scanner_node.h

```diff
@@ -131,15 +142,16 @@ void ROSScannerNodeT<S>::laserScanCallback(const LaserScan& scan)
 {
   try
   {
-    const auto laser_scan_msg = toLaserScanMsg(scan, tf_prefix_, x_axis_rotation_);
+    std::string tf_prefix_with_subscriber_ = tf_prefixes_.at(scan.scannerId());
```

Comment:
Underscore at the end of the name is only used for class members. I would reduce this variable to `std::string tf_prefix`.